geekelo/gft-hackathon-webapp-project
# Milestone: Deploy to make project accessible across the internet

Things worked on;

- Test project for bugs 
- Test pages, functions, etc to check responsiveness on mobile and desktop screen sizes 
- Verify that all errors are fixed
- Deploy project using Github pages
- Include the LIVE DEMO link in the Readme file
- Update Documentation file
